springdoc-openapi-common/src/main/java/org/springdoc/core/SpringDocConfiguration.java

```diff
@@ -39,23 +39,11 @@ public OpenAPIBuilder openAPIBuilder(Optional<OpenAPI> openAPI, ApplicationConte
         return new OpenAPIBuilder(openAPI, context, securityParser,modelConverters);
     }
 
-    @Bean
```

Comment:
Unfortunately had to move creation of these beans to `SpringDocWebMvcConfiguration`+`SpringDocWebFluxConfiguration`, as it was not possible to disable the latter